Presentation.Web/Controllers/External/V2/ItSystemV2Controller.cs

```diff
@@ -84,54 +84,177 @@ public ItSystemV2Controller(IItSystemService itSystemService, IAuthorizationCont
         /// <param name="uuid">Specific IT-System UUID</param>
         /// <returns>Specific data related to the IT-System</returns>
         [HttpGet]
-        [Route("{uuid}")]
+        [Route("it-systems/{uuid}")]
         [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ItSystemResponseDTO))]
         [SwaggerResponse(HttpStatusCode.BadRequest)]
         [SwaggerResponse(HttpStatusCode.Unauthorized)]
         [SwaggerResponse(HttpStatusCode.Forbidden)]
         [SwaggerResponse(HttpStatusCode.NotFound)]
         public IHttpActionResult GetItSystem(Guid uuid)
         {
+            if (uuid == Guid.Empty)
+                return BadRequest(nameof(uuid) + " must be a non-empty guid");
+
             return _itSystemService
                 .GetSystem(uuid)
-                .Select(ToDTO)
+                .Select(ToSystemResponseDTO)
                 .Match(Ok, FromOperationError);
         }
 
-        private static ItSystemResponseDTO ToDTO(ItSystem arg)
+        /// <summary>
+        /// Returns active IT-Systems
+        /// </summary>
+        /// <param name="page">Page index to be returned (zero based)</param>
+        /// <param name="pageSize">Page size</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("rightsholder/it-systems")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(IEnumerable<RightsHolderItSystemResponseDTO>))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        public IHttpActionResult GetItSystemsByRightsHoldersAccess([FromUri] StandardPaginationQuery paginationQuery = null)
```

Comment:
Det vil ikke give mening at kunne sortere så man kan få alle ItSystems for bestemt RightsHolderOrg?

Reply:
I så fald skal der laves en test der også fanger dette scenarie

Reply:
Tja det er i hvert fald ikke aftalt således lige p.t. og det vil kun gavne de brugere der agerer rettighedshavere i flere virksomheder. Det er muligt men spørgsmålet er om det er værd at kaste tid efter, eftersom det ikke er en del af scope lige p.t.